src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+/* #region-page */
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--0 .stars__star:nth-last-child(n + 6),
+.stars--1 .stars__star:nth-last-child(n + 5),
+.stars--2 .stars__star:nth-last-child(n + 4),
+.stars--3 .stars__star:nth-last-child(n + 3),
+.stars--4 .stars__star:nth-last-child(n + 2),
+.stars--5 .stars__star:nth-last-child(n + 1) {
+  background-image: url(./images/star-active.svg);
```

Comment:
Ensure that the image paths './images/star.svg' and './images/star-active.svg' are correct and that the images exist in the specified directory. This is crucial for the stars to display correctly.